monitor.py

```diff
@@ -880,43 +881,54 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
             elif event_type in ('BackPack', 'Backpack'):  # WORKAROUND 4.0.0.200: BackPack becomes Backpack
                 # TODO: v31 doc says this is`backpack.json` ... but Howard Chalkley
                 #       said it's `Backpack.json`
-                with open(join(self.currentdir, 'Backpack.json'), 'rb') as backpack:  # type: ignore
-                    try:
-                        # Preserve property order because why not?
-                        entry = json.load(backpack, object_pairs_hook=OrderedDict)
+                backpack_file = pathlib.Path(str(self.currentdir)) / 'Backpack.json'
+                backpack_data = None
```

Comment:
Something about the 'exception' being on the else irks me.  Invert conditionals in this to log in them, and ... well we're in this huge conditional tree so we can't return :/.